pkg/binding/asker/survey_asker.go

```diff
@@ -61,3 +110,34 @@ func (o *Survey) AskBindAsFiles() (bool, error) {
 	}
 	return bindAsFile, nil
 }
+
+func (o *Survey) SelectCreationOptions() ([]CreationOption, error) {
+	options := []int{}
+	prompt := &survey.MultiSelect{
+		Message: "Check(with Space Bar) one or more operations to perform with the ServiceBinding:",
+		Options: []string{"create it on cluster", "display it", "save it to file"}, // respect order of CreationOption constants
```

Comment:
> // respect order of CreationOption constants

Wouldn't using string constants instead of `iota` make it simpler? That way we could reuse them in these `Options` slice as well, IMO.  WDYT?

Reply:
This is not really a request for change. In case you feel it would be better to use `string`, just remove the lgtm flag, and I'll re-apply it when you push the change. Otherwise, LGTM.

/lgtm

Reply:
I generally prefer using codes (here iota representing the index in the array of proposed options), instead of passing literal texts around. Mainly because literal text is intended to be translated and that would need more manipulation, and also because if is simpler to compare codes instead of literal text.